src/type/directives.js

```diff
@@ -52,14 +66,16 @@ export const GraphQLIncludeDirective = new GraphQLDirective({
   description:
     'Directs the executor to include this field or fragment only when ' +
     'the `if` argument is true.',
+  locations: [
```

Comment:
I made a bit of research (probably should have done from the beginning :) ) and turns out that this query:

```
query Q {
  a
  ...Frag
}
fragment Frag on TestType @skip(if: true) {
  b
}
```

is indeed invalid query, at least it will not pass the query validation. Tests are green because `execute` (which is used in this test) does not validate the queries and executes them directly. Execution engine on the other hand still supports `include` and `skip` directives on fragment definitions, even though they are no longer allowed to be used on fragment definitions.

@leebyron can you please confirm this? Is it intended behavior? I feel that it's not 100% safe to interpret directives if they are used in invalid locations. In my case I decided to completely exclude fragments definitions which use `include` and `skip` directives, regardless of `if` argument (semantically in this case I consider the whole fragment definition to be broken/invalid because of the incorrect usage of the directive):

https://github.com/sangria-graphql/sangria/blob/master/src/test/scala/sangria/execution/DirectivesSpec.scala#L254-L304